data/json/items/gun/shot.json

```diff
@@ -766,4 +766,32 @@
     "flags": [ "RELOAD_ONE" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "shot": 6 } } ]
   }
+  {
+    "id": "makeshift_shotgun",
+    "copy-from": "shotgun_base",
+    "type": "GUN",
+    "name": { "str": "makeshift shotgun" },
+    "description": "A crude shotgun, composed of two thick steel pipes, an end cap and a nail. The lack of sights make this weapon only useful at point-blank range.",
+    "weight": "3629 g",
+    "volume": "2264 ml",
```

Comment:
Seems like there should be some compression here, given one fits at least partially into the other.

Reply:
Any suggestions?

"A crude shotgun composed of two thick steel pipes, an end cap and a nail. It is only useful at point blank range." it my slightly shortened version.